### ISSUE TITLE
Text styles add colours, which convey meaning not available to non-visual or colour blind users.

### ISSUE THREAD

**Issue by msayoung** [localgovdrupal/localgov_base#563]
Our default text styles offer these options:

- Alert info , warning, failure, success
- Callout danger, callout success, (and others) 

These add a coloured outline or background making these stand out visually. The colours also suggest meaning (green - good, red - danger). 

There is no text alternative for these styles meaning that the text won't stand out to non-visual (eg screen reader) users. Colour blind users may not be able to distinguish one from the other. 

This could (or should ? ) be dealt with in content - for example adding "Warning:" to the beginning of the text. 

Thoughts? 

Related https://github.com/localgovdrupal/localgov_base/issues/531

**Comment by markconroy** [comment]
There's [a grand auld conversation going on over here](https://github.com/localgovdrupal/localgov_base/pull/566) along with a nice little PR.
